controllers/repository_test.go

```diff
@@ -16,7 +16,8 @@ func TestCreateRepository(t *testing.T) {
 	defer ctrl.Finish()
 
 	mockECR := mock.NewMockECRAPI(ctrl)
-	controller := NewRepositoryController(mockECR)
+	mockAccountManager := mock.NewMockAccountManager(ctrl)
+	controller := NewRepositoryController(mockECR, mockAccountManager)
 
 	validateCreateRepositoryInput := func(input *ecr.CreateRepositoryInput) {
```

Comment:
[nit] While we're here - can we change this to be consistent with the other `gomock` input patterns?
```go
mockECR.EXPECT().
    CreateRepository(input).
    Return(...)
```

Reply:
✅